client/my-sites/plans-features-main/index.jsx

```diff
@@ -29,8 +29,8 @@ import purchasesPaths from 'me/purchases/paths';
 
 class PlansFeaturesMain extends Component {
 
-	isJetpackSite( site ) {
-		return site.jetpack;
+	isJetpackSite( site, isSiteAutomatedTransfer ) {
+		return ! isSiteAutomatedTransfer && site.jetpack;
```

Comment:
How about we instead pass through a more descriptive prop like: `displayJetpackPlans` we can then remove this function and the extra `isSiteAutomatedTransfer` prop.